lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -436,28 +428,41 @@ func convertContainerToContainerInfo(container *exec.ContainerInfo) *models.Cont
 		}
 	}
 
-	path := container.ExecConfig.Sessions[ccid].Cmd.Path
-	info.ProcessConfig.ExecPath = &path
+	// in heavily loaded environments we were seeing a panic due to a missing
+	// session id in execConfig -- this has only manifested itself in short lived containers
+	// that were initilized via run
```

Comment:
We can compromise by protecting against the panic if DebugLevel == 0.

Irrespective, we should log the actual contents - it's possible the entire extraconfig wasn't populated/unmarshalled correctly.